content/blog/2020-05-19-remote-roundup/index.markdown

```diff
@@ -0,0 +1,115 @@
+
+## [Supervised Machine Learning Case Studies with R](https://supervised-ml-course.netlify.app/)
+
+[<img src="supervised-ml-course.png" title="https://supervised-ml-course.netlify.com/" alt="https://supervised-ml-course.netlify.com/" style="text-align: center" width="500"/>](https://supervised-ml-course.netlify.app/)
+
+This self-paced course is *newly* updated to use the tidymodels framework for predictive modeling, brought to you by [Julia Silge](https://juliasilge.com/). Julia is a member of the tidymodels software engineering team, an RStudio [certified instructor](https://education.rstudio.com/trainers/people/silge+julia/), and prolific data science blogger (don't miss her [\#TidyTuesday](https://juliasilge.com/category/tidymodels/) posts with video screencasts!). When I first started my own [machine learning journey](https://alison.rbind.io/post/2019-12-23-learning-to-teach-machines-to-learn/), Julia's course was a welcome break for me from predicting home prices in [Ames, Iowa](https://cran.r-project.org/web/packages/AmesHousing/index.html)! Julia has a talent for explaining complicated things in simple terms, and the whole course is a breath of fresh air if you've been trying to learn machine learning by wading through stuffy textbooks. I was lucky enough to get a preview of this revised version, and wholeheartedly recommend---it is a great interactive compliment to [tidymodels.org](https://www.tidymodels.org/ "⌘+Click to follow link").
+
+## [Applied Machine Learning](https://www.cs.columbia.edu/~amueller/comsw4995s20/)
+
+[<img src="aml.png" title="https://www.cs.columbia.edu/~amueller/comsw4995s20/" alt="https://www.cs.columbia.edu/~amueller/comsw4995s20/" width="500"/>](https://www.cs.columbia.edu/~amueller/comsw4995s20/)
+
+[Scikit-learn](https://scikit-learn.org) developer and Columbia University Associate Research Scientist [Andreas Mueller](http://amueller.github.io/) has shared his 2020 Applied Machine Learning course. This includes a [schedule](https://www.cs.columbia.edu/~amueller/comsw4995s20/schedule/ "⌘+Click to follow link") with links to all his slides (including presenters notes!), and 21 videotaped lectures in a [YouTube playlist](https://www.youtube.com/playlist?list=PL_pVmAaAnxIRnSw6wiCpSvshFyCREZmlM "⌘+Click to follow link"). Obviously, this is not an R course; it is in Python 🐍. Regardless, any data scientist can benefit from these materials, and it may give you R lovers an excuse to test out using [Python](https://rstudio.com/solutions/r-and-python/) and the [reticulate package](https://rstudio.github.io/reticulate/).
```

Comment:
Why include "Click to follow link"? And why the apple symbol? (We have readers on Windows)

Reply:
This is an artifact of me testing a new IDE feature. Thanks- this is a :bug: